tests/foreman/api/test_convert2rhel.py

```diff
@@ -0,0 +1,323 @@
+"""Test class for converting to RHEL from API
+
+:Requirement: Convert2rhel
+
+:CaseAutomation: Automated
+
+:CaseLevel: Integration
+
+:CaseComponent: Convert2rhel
+
+:Assignee: shwsingh
+
+:TestType: Functional
+
+:Upstream: No
+"""
+import pytest
+import requests
+from broker import VMBroker
+from nailgun import entities
+
+from robottelo import manifests
+from robottelo import ssh
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.api.utils import promote
+from robottelo.api.utils import upload_manifest
+from robottelo.api.utils import wait_for_tasks
+from robottelo.config import settings
+from robottelo.constants import DEFAULT_ARCHITECTURE
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.constants import REPOS
+from robottelo.hosts import ContentHost
+
+
+def create_repo(org, repo_url, ssl_cert=None):
+    """Create and sync repository"""
+    product = entities.Product(organization=org).create()
+    options = {
+        'organization': org,
+        'product': product,
+        'content_type': 'yum',
+        'url': repo_url,
+        'ssl_ca_cert': ssl_cert,
+        'unprotected': True,
+        'verify_ssl_on_sync': False,
+    }
+    repo = entities.Repository(**options).create()
+    repo.product = product
+    repo.sync()
+    return repo
+
+
+def create_activation_key(org, lce, cv, subscription_id):
+    """Create activation key with subscription"""
+    act_key = entities.ActivationKey(
+        organization=org,
+        content_view=cv,
+        environment=lce,
+    ).create()
+    act_key.add_subscriptions(data={'subscription_id': subscription_id})
+    return act_key
+
+
+def update_cv(cv, lce, repos):
+    """Update and publish Content view with repos"""
+    cv = entities.ContentView(id=cv.id, repository=repos).update(["repository"])
+    cv.publish()
+    cv = cv.read()
+    promote(cv.version[-1], environment_id=lce.id)
+    return cv
+
+
+def register_host(act_key, module_org, module_loc, hostname, ubi=None):
+    """Register host to satellite"""
+    # generate registration command
+    command = entities.RegistrationCommand(
+        organization=module_org,
+        activation_keys=[act_key.name],
+        location=module_loc,
+        insecure=True,
+        repo=ubi,
+    ).create()['registration_command']
+    result = ssh.command(command, hostname=hostname)
+    assert result.status == 0
+
+
+@pytest.fixture(scope="module")
+def ssl_cert(module_org):
+    """Create credetial with SSL cert for Oracle Linux"""
+    res = requests.get(settings.repos.convert2rhel.ssl_cert_oracle)
+    res.raise_for_status()
+    return entities.ContentCredential(
+        content=res.text, organization=module_org, content_type='cert'
+    ).create()
+
+
+@pytest.fixture(scope='module')
+def manifest(module_org):
```

Comment:
there already is a fixture in taxonomies named `module_manifest_org`, but it is creating a new org instead of using `module_org` which is used in my other fixtures. This org with manifest is used a lot, so I didn't want to change it to use `module_org` because I am not sure what it could cause.